tracer-core/src/main/java/org/zalando/tracer/DockerGenerator.java

```diff
@@ -0,0 +1,421 @@
+package org.zalando.tracer;
+
+/*
+ * ⁣​
+ * Tracer
+ * ⁣⁣
+ * Copyright (C) 2015 Zalando SE
+ * ⁣⁣
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * 
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * ​⁣
+ */
+import com.google.common.annotations.VisibleForTesting;
+import java.util.UUID;
+
+public final class DockerGenerator implements Generator {
```

Comment:
This is kind of easter egg or joke, but we can think is it possible to make it useful.
The main requirement that is should be human readable, compatible and memorable.
To improve it maximum number of different ids before repetition should be defined.